src/app/pages/dashboard/services/widget-resources.service.ts

```diff
@@ -96,6 +97,8 @@ export class WidgetResourcesService {
   );
 
   networkInterfaceLastHourStats(interfaceName: string): Observable<ReportingData[]> {
+    this.store$.dispatch(systemInfoLoaded({ systemInfo: null }));
+    this.store$.dispatch(systemInfoUpdated());
```

Comment:
I think it's kinda weird that a method called `networkInterfaceLastHourStats` makes changes like this.